core/shared/platform/include/platform_api_vmcore.h

```diff
@@ -65,6 +65,12 @@ os_vprintf(const char *format, va_list ap);
  */
 uint64
 os_time_get_boot_microsecond(void);
+/* Get current clock time*/
```

Comment:
because the function doesn't return the clock time, it's worth documenting that it returns BHT_OK on success and BHT_ERROR on failure. The same for res_get function.